qa/rpc-tests/README.md

```diff
@@ -1,10 +1,8 @@
 Regression tests
 ================
 
-### [python-bitcoinrpc](https://github.com/jgarzik/python-bitcoinrpc)
-Git subtree of [https://github.com/jgarzik/python-bitcoinrpc](https://github.com/jgarzik/python-bitcoinrpc).
-Changes to python-bitcoinrpc should be made upstream, and then
-pulled here using git subtree.
+### [test_framework](test_framework)
+This contains some files of the [python-bitcoinrpc by jgarzik](https://github.com/jgarzik/python-bitcoinrpc).
```

Comment:
Only some files are taken from python-bitcoinrpc, I think the python test-framework code was originally written by @gavinandresen (https://github.com/bitcoin/bitcoin/pull/4485). IMO we should remove the description for the `test_framework` folder itself.